### ISSUE TITLE
Tooltip crashing on accounts with many items rendered

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#119]
```
Uncaught RangeError: Maximum call stack size exceeded
    at r.fn.init.push (<anonymous>)
    at Function.ga [as find] (jquery-3.2.1.min.js:2)
    at r.fn.init.find (jquery-3.2.1.min.js:2)
    at new r.fn.init (jquery-3.2.1.min.js:2)
    at r (jquery-3.2.1.min.js:2)
    at Object.setuptools.app.muledump.tooltip (muledump.js:261)
    at update_totals (totals.js:143)
    at option_updated (options.js:180)
    at HTMLInputElement.<anonymous> (options.js:125)
    at HTMLInputElement.dispatch (jquery-3.2.1.min.js:3)
    at HTMLInputElement.q.handle (jquery-3.2.1.min.js:3)
```

**Comment by jakcodex** [comment]
This issue is known to exist in two locations:

1. Tooltip will always crash if the number of items exceeds jquery's limitations. This can be resolved by binding tooltip to each item directly rather than once-for-all as it is now.

2. Counting items as a whole also breaks. This can be resolved by counting mules individually and adding up their totals.

**Comment by jakcodex** [comment]
Now bound to each item individually instead of trying to select all items on screen for tracking.

**Comment by jakcodex** [comment]
A fix for item 1 (specifically relating to tooltip) is included in 9.1.

The second issue is more complicated. JQuery is unable to select all items if there are too many on screen. But selecting with document.querySelectorAll('.item') works just fine. This is a bug in jQuery it seems. Will need to spend more time looking for a solution. This will not be in 9.1.

**Comment by jakcodex** [comment]
A somewhat happy middle ground is presently available in v9.3.0. If the user disables items from display in Options but enables Global Totals, then they can still see the totals and item tooltip. Not rendering the items on mules will relieve the excess jquery calls leading to the failure.
